src/index.html

```diff
@@ -17,6 +17,84 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 class="heading">Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="Logo"
+          class="logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#apple"
+              class="is-active"
+            >
+              Apple
+            </a>
```

Comment:
Add classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#apple"
              class="nav__link is-active"
            >
              Apple
            </a>
```